[ADD] sms marketing: added essentials/getting started docs (#2553)
odoo/documentation: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst

the main :guilabel:`Mailing Lists` page, and Odoo reveals the mailing list detail form.

Along the top of the mailing list detail form, there are various analytical smart buttons displaying
statistics for different metrics related to the mailing list (e.g. :guilabel:`Recipients`,
:guilabel:`Mailings`, etc.).

To take a closer look at (or make changes to) any of those elements, click the desired smart button,
and Odoo reveals a separate page with in-depth data associated with the mailing list.

To make changes to the mailing list itself, click the :guilabel:`Edit` button in the upper-left
corner of the mailing list detail form.

.. image:: mailing_lists_blacklists/sms-mailing-list.png
   :align: center
   :alt: View of the mailing list template form in Odoo SMS Marketing.

.. note::
   Don't forget to hit the :guilabel:`Save` button once changes have been made.

Mailing lists contacts
======================

If :guilabel:`Mailing List Contacts` is selected from the :guilabel:`Mailing Lists` header drop-down
menu, Odoo reveals a separate page with all the contacts associated with one (or more) of the

> configured mailing lists in the database.

Reviewer: Rewording to remove passive/conditional tense in favor of active.
```suggestion
Access contacts information from one or more mailing lists navigate to :menuselection:`Mailing Lists
--> Mailing List Contacts` to reveal a dashboard with with all the contacts associated with one or
more of the configured mailing lists in the database.
```

.. image:: mailing_lists_blacklists/mailing-list-contacts-page.png
   :align: center
   :alt: View of the mailing lists contact page in the Odoo SMS Marketing application.

.. note::
   By default, Odoo reveals the :guilabel:`Mailing List Contacts` page with the :guilabel:`Exclude
   Blacklisted Phone` filter in the search bar. Therefore only showing contact information for
   recipients who still want to receive communications and mailings.

Communication history in the Chatter
------------------------------------

An accessible record of the sent mailings is kept on each recipient's :guilabel:`Chatter`, located

> beneath a recipient's contact form (in the :guilabel:`Contacts` application).

Reviewer: Don't think these get guilabels since it's not active instruction on the UI.
```suggestion
An accessible record of every sent mailing(s) is kept on each recipient's *chatter* section, located
beneath a recipient's contact form (in the *Contacts* application).
```

Database users can reference the :guilabel:`Chatter` to easily keep track of communications, and see
a history of interactions with contacts and prospects.

For example, sales representatives can use the :guilabel:`Chatter` to quickly find out which SMS

> mailing promotions a certain customer has received (or not received).

Reviewer: Same here.
When referencing BE docs they don't use guilabels or capitalize this term.
We can mirror that style, for now, and can clarify at some point.
```suggestion
Database users can reference the chatter to easily keep track of communications, and see a history
of interactions with contacts and prospects.

For example, sales representatives can use the chatter to quickly find out which :abbr:`SMS (Short
Message Service)` mailing promotions a certain customer has received (or not received).
```

.. image:: mailing_lists_blacklists/sms-marketing-chatter.png
   :align: center
   :alt: View of what the chatter looks like in the Odoo Contacts app.

Blacklist
=========

> :guilabel:`Odoo SMS Marketing` has a :guilabel:`Blacklist` feature that provides recipients with the

Reviewer: No guilabel here
```suggestion
Odoo *SMS Marketing* has a :guilabel:`Blacklist` feature that provides recipients with the power to
```

power to add their phone number to a list of people who no longer want to receieve communications or
mailings.

This is also known as the "Unsubscribe Process." Customers will automatically be added onto the
:guilabel:`Blacklist`, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription

> Management Page`. Customers can also be added manually to the blacklist, if necessary.

Reviewer: Formatting
```suggestion
This is also known as the unsubscribe process: customers will automatically be added onto a
*blacklist*, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription Management`
page. Customers can also be added manually to the blacklist, if necessary.
```

To see a complete collection of blacklisted numbers, navigate to the :guilabel:`SMS Marketing` app,
and go to :menuselection:`Configuration --> Blacklisted Phone Numbers`. Doing so, reveals a separate

> page containing every blacklisted phone number in the database.

Reviewer: Wording/formatting.
```suggestion
To see a complete collection of blacklisted numbers, navigate to the :menuselection:`SMS Marketing
app --> Configuration --> Blacklisted Phone Numbers` to reveal a dashboard containing every
blacklisted phone number in the database.
```

.. image:: mailing_lists_blacklists/sms-blacklist-menu.png
   :align: center
   :alt: SMS Blacklist menu in the application.

To manually add a number, click the :guilabel:`Create` button in the upper-left. When clicked, Odoo
reveals a separate page, in which the phone number to be blacklisted is entered. There's also a

> checkbox to indicate whether that particular phone numnber is :guilabel:`Active` (or not).

Reviewer: "Odoo reveals a separate page" phrase is too wordy. Just say what's happening in clear terms 😉 
```suggestion
To manually add a number to a blacklist, click the :guilabel:`Create` button in the upper-left
corner of the dashboard and enter the phone number on the next page's form. There's also a
```

.. image:: mailing_lists_blacklists/sms-blacklist-create.png
   :align: center
   :alt: SMS Blacklist menu in the application.

Once the form is completed, click :guilabel:`Save` to add it to the :guilabel:`Blacklist`. To
remove any number from the :guilabel:`Blacklist`, select the desired number, and click

> :guilabel:`Unblacklist`.

Reviewer: `Blacklisted Phone Numbers` gets a guilabel because it's a menu item and the name of a dashboard, while _blacklist_ does not. Added some more specificity here too.

```suggestion
Once the form is completed, click :guilabel:`Save` to add it to the :guilabel:`Blacklisted Phone
Numbers` list. To remove any number from the blacklist, select the desired number on the dashboard,
and then, on the phone number's form, click :guilabel:`Unblacklist`.
```

Importing blacklists
--------------------

During a software/platform migration, it is possible to import an already existing blacklist of

> contacts. This would include customers, who have already asked to be :guilabel:`Blacklisted` on SMS

Reviewer: ```suggestion
contacts. This would include customers, who have already asked to be blacklisted` on :abbr:`SMS
(Short Message Service)` mailings.
```

mailings.

To do that, navigate to the :guilabel:`SMS Marketing` app, and go to :menuselection:`Configuration
--> Blacklisted Phone Numbers`. Then, select the :guilabel:`Favorites` drop-down (beneath

> the search bar), and click :guilabel:`Import records`.

Reviewer: ```suggestion
To do that, navigate to :menuselection:`SMS Marketing app --> Configuration --> Blacklisted Phone
Numbers`, and then select the :guilabel:`Favorites` drop-down menu (beneath the search bar), and
click :guilabel:`Import records`.
```
